ceph.spec.in

```diff
@@ -813,18 +816,42 @@ export CPPFLAGS="$java_inc"
 export CFLAGS="$RPM_OPT_FLAGS"
 export CXXFLAGS="$RPM_OPT_FLAGS"
 
-env | sort
-
+# Parallel build settings ...
+# unlimit _smp_mflags in system macro
+%global _smp_ncpus_max 0
+CEPH_MFLAGS_JOBS="%{?_smp_mflags}"
 %if %{with lowmem_builder}
-%if 0%{?jobs} > 8
-%define _smp_mflags -j8
-%endif
+lo_jobs="%{?jobs:%{jobs}}"
+%if 0%{?suse_version}
+%ifarch %arm
+    %if ! 0%{?qemu_user_space_build:1}
+        # native hardware is not that powerful
+        lo_jobs="1"
+    %endif
+%endif
+%endif
+# do not eat all memory
+echo "Available memory:"
+free
+echo "System limits:"
+ulimit -a
+lo_jobs_reduced=
+if test -n "$lo_jobs" -a "$lo_jobs" -gt 1 ; then
+    mem_per_process=1600
+    max_mem=$(LANG=C free -t -m | sed -n "s|^Mem: *\([0-9]*\).*$|\1|p")
```

Comment:
Why do you need the -t flag when you are sedding Mem: anyway?

Reply:
got rid of `-t`